src/master/filesystem_checksum.cc

```diff
@@ -35,6 +35,7 @@ static uint64_t fsnodes_checksum(FSNode *node, bool full_update = false) {
 		return 0;
 	}
 	uint64_t seed = 0x4660fe60565ba616;  // random number
+	uint64_t lowerCaseSeed = 0x4660fe60565ba616;  // random number for lower case
```

Comment:
Why have them different if they are the same seed?

Reply:
Done